lib/board.rb

```diff
@@ -0,0 +1,66 @@
+require "pry"
+class Board
+  attr_accessor :cells
+  attr_reader :display
+  
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.